zeg/tests/test_config.py

```diff
@@ -52,9 +55,14 @@ def test_unknown_configuration(self):
         """
 
         configuration = self._get_configuration(config_data)
-
-        with self.assertRaises(exceptions.ValidationError):
-            config.validate_config(configuration)
+       
+        with self.assertRaises(SystemExit):
+            config.validate_config(configuration, logger)
+        
+        logger.error.assert_called_with(
```

Comment:
Rather than mocking the logger, use [`assertLogs()`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertLogs) context manager which handles the capturing for you.

Reply:
If i am not mistaken our `log.py` does not implement 

https://docs.python.org/3/library/logging.html#logging.Logger

Reply:
Hm, probably a change for later then. Better to test with a stream or line list rather than mocking, but can do that later.